src/app/components/ATIAnalytics/types.ts

```diff
@@ -161,6 +161,21 @@ export interface ATIEventTrackingProps {
   experimentVariant?: string;
   ampExperimentName?: string;
   preventNavigation?: string;
+  componentSpecificTrackers?: ComponentSpecificTrackers;
+  blockSpecificTrackers?: BlockSpecificTrackers;
```

Comment:
Could these be singular instead of plural?
```suggestion
  componentSpecificTracker?: ComponentSpecificTracker;
  blockSpecificTracker: BlockSpecificTracker;
```